docs/configuring.md

```
@@ -238,6 +238,34 @@ liveness detection. Default: `300` (5 minutes).
+
+```
+SET spock.sync_timeout = '2h';
+CALL spock.add_node(...);
+```
+
+This bounds a whole wait, not an individual probe of a remote node. Routines
+that additionally bound each remote probe - so that one unresponsive peer
+cannot consume the entire budget - keep their own, much smaller limit for that
+purpose.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)